stixcore/tmtc/packet_factory.py

```diff
@@ -117,9 +115,9 @@ class ValidationFunctionError(AttributeError):
     """
 
 
-if 'pytest' in sys.modules:
-    GenericPacket.idb_manager = IDBManager(test_data.idb.DIR)
-else:
-    GenericPacket.idb_manager = IDBManager(Path(__file__).parent.parent / "data" / "idb")
+# if 'pytest' in sys.modules:
+#     GenericPacket.idb_manager = IDBManager(test_data.idb.DIR)
+# else:
+GenericPacket.idb_manager = IDBManager(Path(__file__).parent.parent / "data" / "idb")
 
 Packet = TMTCPacketFactory(registry=GenericPacket._registry)
```

Comment:
i do not see where we set the other dir in case we are running tests.

Reply:
Need to manually set it, what if I've imported pytest but am not running tests or running test in a different part of the code? The idea would be in the test setup you would explicitly define these
```
GenericPacket.idb_manager = IDBManager(test_data.idb.DIR)
Packet = TMTCPacketFactory(registry=GenericPacket._registry)
```